src/literature/components/AuthorLink.jsx

```diff
@@ -1,21 +1,40 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { Tooltip } from 'antd';
+import { Map } from 'immutable';
 
 class AuthorLink extends Component {
+  getAuthorHref() {
+    const { author, recordId } = this.props;
+    let href = `//inspirehep.net/author/profile/${author.get('full_name')}`;
+    if (recordId != null) {
```

Comment:
``!==`` :P

Reply:
shouldn't be undefined? https://github.com/inspirehep/inspirehep-ui/pull/14/files#diff-9921d7d631405e36feb211d5e67c07e4R37

Reply:
`=! null` is OK to use actually. It is checking if `null` or `undefined`

Reply:
I wanted to check for null also, because if somehow it is set to `null`, then `defaultProp` won't be used and code might break :(